RoyalCar/OccupyException.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Prestige.RoyalCar
+{
+    class OccupyException : ApplicationException
```

Comment:
please see official microsoft documentation: https://docs.microsoft.com/en-us/dotnet/api/system.applicationexception?view=netframework-4.7.2

use Exception class to derive from